ui/app/AppLayouts/Chat/controls/UserDelegate.qml

```diff
@@ -126,6 +123,8 @@ Item {
                     messageContextMenu.selectedUserDisplayName = wrapper.name
                     messageContextMenu.selectedUserIcon = wrapper.iconToShow
                     messageContextMenu.popup()
```

Comment:
Should we prevent the context menu from dimming the entire app?
We have this cause it's the default, but AFAIK we don't have it in the designs and maybe it makes sense to turn it off consistently...

Anyways, just spitballing

Reply:
I can remove it, sure. Is the popup dimming still fine though?

Reply:
I personally don't mind the popups doing that. But maybe that's something we should generally check with design and @John-44

Reply:
agree, maybe it's also a mistake in design. so I'll wait for design input. thanks for bringing this up!

Reply:
> Should we prevent the context menu from dimming the entire app?

I agree, dimming the app is ok for popups, but not for displaying context menus, (but I am not designer, that's only my opinion :) ).